### PR TITLE
🎉  add echtvar 0.1.9

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Part of https://github.com/d3b-center/bixu-tracker/issues/2108

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Built locally and tested it here: https://github.com/d3b-center/bixu-tracker/issues/2108

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: echtvar/0.1.9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM staphb/bcftools:1.18
```
Comment (migbro) [inline echtvar/0.1.9/Dockerfile:1]: Might be good to also add the label maintainer and description

**Review by migbro — APPROVED** [review]
Seems good in general, we might want to make part of our SOP having a maintainer and description label

File: echtvar/0.1.9/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+RUN apt update && apt install -y wget
+
+RUN wget -O /bin/echtvar https://github.com/brentp/echtvar/releases/download/v0.1.9/echtvar \
```
Comment (sakshamphul) [inline echtvar/0.1.9/Dockerfile:8]: Perhaps, add a version tag on the top of the file (~v0.1.9)

File: echtvar/0.1.9/Dockerfile (hunk 1 above)
Comment (dmiller15) [reply echtvar/0.1.9/Dockerfile:1]: Done

File: echtvar/0.1.9/Dockerfile (hunk 2 above)
Comment (migbro) [reply echtvar/0.1.9/Dockerfile:8]: Ah yeah the `ENV VERSION=0.1.9` syntax, then ${VERSION} in the wget command

Comment (dmiller15) [reply echtvar/0.1.9/Dockerfile]: Good catch!

**Review by sakshamphul — APPROVED** [review]

**Pull request merged**